Pod/Classes/API/ABMAuth2Manager.h

```diff
@@ -0,0 +1,53 @@
+//
+//  ABMAuth2Manager.h
+//  Pods
+//
+//  Created by Benjamin Maer on 4/6/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+
+
+
+
+@class ABMAuthenticationCredentials;
+
+
+
+
+
+typedef void(^abm_Auth2Manager_authenticationSuccessBlock) (ABMAuthenticationCredentials *credentials);
```

Comment:
Maybe there is no need of using underscore notation. I would use just `Auth2ManagerAuthenticationSuccessBlock`. And the same for successBlock and failureBlock.